src/GatewayAPI/Configuration/NetworkGatewayConfiguration.cs

```diff
@@ -126,4 +128,8 @@ public AcceptableLedgerLag GetAcceptableLedgerLag()
 
         return acceptableLedgerLag;
     }
+
+    public int GetMaxPageSize() {
+        return _configuration.GetValue("MaxPageSize", 30);
```

Comment:
As github says, the braces should be on a separate line to the method in C# style.